hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableOutputFormat.java

```diff
@@ -307,17 +307,11 @@ public static CarbonLoadModel getLoadModel(Configuration conf) throws IOExceptio
     }
     model.setDateFormat(
         conf.get(
-            DATE_FORMAT,
-            carbonProperty.getProperty(
-                CarbonLoadOptionConstants.CARBON_OPTIONS_DATEFORMAT,
-                CarbonLoadOptionConstants.CARBON_OPTIONS_DATEFORMAT_DEFAULT)));
+            DATE_FORMAT, ""));
```

Comment:
I thinks should take default fimestamp if  not set in conf, please confirm

Reply:
Fixed